yarn-project/aztec.js/src/contract/deploy_account_sent_tx.ts

```diff
@@ -17,11 +16,11 @@ export type DeployAccountTxReceipt = FieldsOf<TxReceipt> & {
  */
 export class DeployAccountSentTx extends SentTx {
   constructor(
-    pxeNodeOrWallet: Wallet | AztecNode | PXE,
+    wallet: Wallet,
```

Comment:
This was always only used with a wallet so I constrained the type just to Wallet. This will help us later on figure out how it all is even used.

Reply:
this was used with a node by externals, can we at least keep the node?

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/17109/commits/50117599bea73fed23be13474d67f9dcbfa8c6df